requirements.txt

```diff
@@ -1,3 +1,6 @@
+google-cloud-spanner==3.49.1
 langchain-core==0.3.9
 langchain-community==0.3.1
-google-cloud-spanner==3.49.1
+langchain-experimental==0.3.2
```

Comment:
if langchain-experimental is not used in colab, it's preferred to not add to requirements.

Reply:
done